sqlstate/__init__.py

```diff
@@ -1,13 +1,13 @@
 from contextlib import asynccontextmanager, contextmanager
 from dataclasses import InitVar, dataclass
 from types import SimpleNamespace
-from typing import Optional
+from typing import Optional, AsyncIterator
 
-import aiopg.sa as aio
 from pydantic import BaseModel, FilePath
 from sqlalchemy import MetaData, Table, create_engine
 from sqlalchemy.engine import Connection, Engine
 from sqlalchemy.engine.url import URL
+from sqlalchemy.ext.asyncio import AsyncConnection, AsyncEngine, create_async_engine
```

Comment:
Sweet :tada:

Reply:
Though maybe we shozld extend it to not make such a breaking change.